modules/sdk-core/src/account-lib/mpc/tss/eddsa/types.ts

```diff
@@ -11,6 +11,7 @@ export interface YShare {
   i: number;
```

Comment:
backward incompatible changes needs to be addressed as `BREAKING` in the commit comments, if this type change will cause incompatibility

Reply:
I really don't know. @sachushaji Do you know if this is a backwards incompatible change?

Reply:
Added breaking change info to commit footer.